tests/spec/core/utils-spec.js

```diff
@@ -453,6 +454,47 @@ describe("Core - Utils", () => {
     });
   });
 
+  describe("htmlJoinAnd", () => {
+    it("joins with proper commas and 'and'", () => {
+      const div = document.createElement("div");
+
+      hyperHTML.bind(div)`${utils.htmlJoinAnd(
+        [],
+        item => hyperHTML`<a>${item}</a>`
+      )}`;
+      expect(div.textContent).toBe("");
+
+      hyperHTML.bind(div)`${utils.htmlJoinAnd(
+        ["x"],
+        item => hyperHTML`<a>${item}</a>`
+      )}`;
+      expect(div.textContent).toBe("x");
```

Comment:
Not just "x", we should try inserting some HTML and check it escapes well. `"<x>"` may work.